no-crime-envelopes-processed-alert.tf

```diff
@@ -0,0 +1,24 @@
+module "no-crime-envelopes-processed-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "No_crime_envelopes_processed_-_Blob_Router"
+  alert_desc = "Triggers when Blob Router did not process single CRIME envelope in last hour within SLA."
+
+  app_insights_query = <<EOF
+traces
+| where message startswith "No Envelopes created in CRIME"
+| where timestamp > now(-70m) and timestamp  < now()
```

Comment:
same as above

Reply:
This will need blob-router code change. I'll raise a pr for that

Reply:
Updated logging in https://github.com/hmcts/blob-router-service/pull/355